src/AzureIoTHubClient.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Arduino. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <time.h>
+#include <sys/time.h>
+
+#include <AzureIoTHubClient.h>
+
+Client* AzureIoTHubClient::sslClient = NULL;
+
+AzureIoTHubClient::AzureIoTHubClient(Client& sslClient)
+{
+    AzureIoTHubClient::sslClient = &sslClient;
```

Comment:
If RTCZero already has this pattern then it is probably OK as is. Since it fits in with the Arduino style.